### PR TITLE
Fix code examples in markdown documentation & Scaladoc

### PR DESCRIPTION
**Pull request by cchantep**
(do not commit there the SBT plugins used to find the errors, only documentation changes)

### CODE REVIEW

File: bench/src/main/scala/play/api/libs/ws/ahc/StandaloneAhcWSRequestBenchMapsBench.scala [hunk 1]
```diff
@@ -26,8 +26,8 @@ import play.api.libs.ws.StandaloneWSRequest
  * Compare your results before/after on your machine. Don't trust the ones in scaladoc.
  *
  * Sample benchmark results:
- * {{{
- * > bench/jmh:run .*StandaloneAhcWSRequestBenchMapsBench
+ *
+ * `> bench/jmh:run .*StandaloneAhcWSRequestBenchMapsBench
```
Comment (renatocaval) [inline bench/src/main/scala/play/api/libs/ws/ahc/StandaloneAhcWSRequestBenchMapsBench.scala]: Is this backtick intentional? Is that a kind of escape?

File: play-ahc-ws-standalone/src/main/scala/play/api/libs/ws/ahc/StreamedResponse.scala [hunk 2]
```diff
@@ -16,11 +16,20 @@ import play.shaded.ahc.org.asynchttpclient.HttpResponseBodyPart
...
- *   }
+ * import scala.concurrent.ExecutionContext
+ *
+ * import akka.util.ByteString
+ * import akka.stream.scaladsl.Source
+ *
+ * import play.api.libs.ws.DefaultBodyReadables._
+ * import play.api.libs.ws.ahc.StandaloneAhcWSClient
+ *
+ * class MyClass(ws: StandaloneAhcWSClient) {
```
Comment (wsargent) [inline play-ahc-ws-standalone/src/main/scala/play/api/libs/ws/ahc/StreamedResponse.scala]: Good call, doesn't need the import

File: play-ahc-ws-standalone/src/main/scala/play/api/libs/ws/ahc/StreamedResponse.scala [hunk 3]
```diff
@@ -16,11 +16,20 @@ import play.shaded.ahc.org.asynchttpclient.HttpResponseBodyPart
...
+ *   def doIt(implicit ec: ExecutionContext) =
```
Comment (wsargent) [inline play-ahc-ws-standalone/src/main/scala/play/api/libs/ws/ahc/StreamedResponse.scala]: This should have an explicit return type to make it clearer

File: play-ws-standalone/src/main/scala/play/api/libs/ws/DefaultBodyReadables.scala [hunk 4]
```diff
@@ -10,51 +10,69 @@ import akka.stream.scaladsl.Source
...
+   * Converts a response body into an `akka.util.ByteString`:
    *
    * {{{
-   * val byteString = response.body[ByteString]
+   * import akka.util.ByteString
+   * import play.api.libs.ws.DefaultBodyReadables._
+   *
+   * def example(response: play.api.libs.ws.StandaloneWSResponse) = {
+   *   val _: ByteString = response.body[ByteString]
+   * }
```
Comment (wsargent) [inline play-ws-standalone/src/main/scala/play/api/libs/ws/DefaultBodyReadables.scala]:
```
   * def example(response: play.api.libs.ws.StandaloneWSResponse): ByteString = {
   *   response.body[ByteString]
   * }
```

File: play-ws-standalone/src/main/scala/play/api/libs/ws/DefaultBodyReadables.scala [hunk 5]
```diff
@@ -10,51 +10,69 @@ import akka.stream.scaladsl.Source
...
    *
    * Note: this is only a best-guess effort and does not handle all content types. See
    * [[StandaloneWSResponse.body:String*]] for more information.
    *
    * {{{
-   * val string = response.body[String]
+   * import play.api.libs.ws.DefaultBodyReadables._
+   *
+   * def example(response: play.api.libs.ws.StandaloneWSResponse) = {
+   *   val _: String = response.body[String]
```
Comment (wsargent) [inline play-ws-standalone/src/main/scala/play/api/libs/ws/DefaultBodyReadables.scala]: same as above

File: play-ws-standalone/src/main/scala/play/api/libs/ws/DefaultBodyReadables.scala [hunk 6]
```diff
@@ -10,51 +10,69 @@ import akka.stream.scaladsl.Source
...
-   * Converts a response body into a read only ByteBuffer.
+   * Converts a response body into a read only `ByteBuffer`.
    *
    * {{{
-   * val buffer = response.body[ByteBuffer]
+   * import java.nio.ByteBuffer
+   * import play.api.libs.ws.DefaultBodyReadables._
+   *
+   * def example(response: play.api.libs.ws.StandaloneWSResponse) = {
+   *   val _: ByteBuffer = response.body[ByteBuffer]
```
Comment (wsargent) [inline play-ws-standalone/src/main/scala/play/api/libs/ws/DefaultBodyReadables.scala]: same as above

File: play-ws-standalone/src/main/scala/play/api/libs/ws/DefaultBodyReadables.scala [hunk 7]
```diff
@@ -10,51 +10,69 @@ import akka.stream.scaladsl.Source
...
   /**
-   * Converts a response body into Array[Byte].
+   * Converts a response body into `Array[Byte]`.
    *
    * {{{
-   * val byteArray = response.body[Array[Byte]]
+   * import play.api.libs.ws.DefaultBodyReadables._
+   *
+   * def example(response: play.api.libs.ws.StandaloneWSResponse) = {
+   *   val _: Array[Byte] = response.body[Array[Byte]]
```
Comment (wsargent) [inline play-ws-standalone/src/main/scala/play/api/libs/ws/DefaultBodyReadables.scala]: same as above

File: play-ws-standalone/src/main/scala/play/api/libs/ws/WSRequestFilter.scala [hunk 8]
```diff
@@ -25,8 +27,10 @@ object WSRequestFilter {
    * Creates an adhoc filter from a function:
    *
    * {{{
+   * import play.api.libs.ws.{ WSRequestFilter, WSRequestExecutor }
+   *
    * val filter: WSRequestFilter = WSRequestFilter { e =>
-   *   WSRequestExecutor(r => e.apply(r.withQueryString("bed" -> "1")))
+   *   WSRequestExecutor(r => e.apply(r.withQueryStringParameters("bed" -> "1")))
```
Comment (wsargent) [inline play-ws-standalone/src/main/scala/play/api/libs/ws/WSRequestFilter.scala]: good call

File: play-ahc-ws-standalone/src/main/scala/play/api/libs/ws/ahc/StreamedResponse.scala (hunk 2 above)
Comment (cchantep) [reply play-ahc-ws-standalone/src/main/scala/play/api/libs/ws/ahc/StreamedResponse.scala]: To make the code example standalone, the import is required

File: play-ws-standalone/src/main/scala/play/api/libs/ws/DefaultBodyReadables.scala [hunk 9]
```diff
@@ -10,51 +10,69 @@ import akka.stream.scaladsl.Source
...
+   * }
    * }}}
    */
   implicit val readableAsByteArray: BodyReadable[Array[Byte]] = BodyReadable(_.bodyAsBytes.toArray)
 
   /**
-   * Converts a response body into Source[ByteString, NotUsed].
+   * Converts a response body into `Source[ByteString, _]`.
```
Comment (cchantep) [inline play-ws-standalone/src/main/scala/play/api/libs/ws/DefaultBodyReadables.scala]: BTW I find the `_` weird

**Comment by cchantep** [comment]
Up @wsargent

File: play-ahc-ws-standalone/src/main/scala/play/api/libs/ws/ahc/StreamedResponse.scala (hunk 2 above)
Comment (wsargent) [reply play-ahc-ws-standalone/src/main/scala/play/api/libs/ws/ahc/StreamedResponse.scala]: yes, agreed

File: play-ws-standalone/src/main/scala/play/api/libs/ws/DefaultBodyReadables.scala (hunk 9 above)
Comment (wsargent) [reply play-ws-standalone/src/main/scala/play/api/libs/ws/DefaultBodyReadables.scala]: me too

**Pull request merged**